<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formwidget</class>
 <widget class="QWidget" name="Formwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UPDATA">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DOWNLOAD">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DOWNLOAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OPENFILE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OPENFILE</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <action name="actionFIXED">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FIXED</string>
   </property>
  </action>
  <action name="actionFLOAT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FLOAT</string>
   </property>
  </action>
  <action name="actionFIXED_FLOAT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FIXED/FLOAT</string>
   </property>
  </action>
  <action name="actionPERADD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>(+%)</string>
   </property>
  </action>
  <action name="actionPERRED">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>(-%)</string>
   </property>
  </action>
  <action name="clerk_actionVoid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>void</string>
   </property>
  </action>
  <action name="clerk_actionCorrect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>correct</string>
   </property>
  </action>
  <action name="clerk_actionCancle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cancle</string>
   </property>
  </action>
  <action name="clerk_actionReturn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>return</string>
   </property>
  </action>
  <action name="clerk_actionDisc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>disc</string>
   </property>
  </action>
  <action name="clerk_actionAdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="clerk_actionAmountdisc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>amountdisc</string>
   </property>
  </action>
  <action name="clerk_actionTrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>train</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
